modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/PlayerHeaderFragment.kt

```diff
@@ -525,13 +525,17 @@ class PlayerHeaderFragment : BaseFragment(), PlayerClickListener {
         viewModel.nextChapter()
     }
 
-    fun onMoreClicked() {
+    fun openMoreMenu(sourceView: SourceView?) {
+        onMoreClicked(sourceView)
+    }
+
+    private fun onMoreClicked(sourceView: SourceView? = null) {
```

Comment:
Could we drop the `openMoreMenu` method and just use `onMoreClicked`?

Reply:
Addressed in https://github.com/Automattic/pocket-casts-android/pull/1478/commits/38fcc5056a7bec1fad7191b79dde7ebafa96d9ce